From 5f5d7fd9bb6ee394256aa23fe5dde8ac35198f64 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 24 Nov 2017 21:33:21 +0100
Subject: [PATCH 1/1] distro bootcmd: define bootloader name for x86

If we want to use distro boot on x86 we need to define the
name of the bootloader EFI application.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/config_distro_bootcmd.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/config_distro_bootcmd.h b/include/config_distro_bootcmd.h
index e0d0034ed3..eb2bdfa39d 100644
--- a/include/config_distro_bootcmd.h
+++ b/include/config_distro_bootcmd.h
@@ -91,6 +91,10 @@
 #define BOOTEFI_NAME "bootaa64.efi"
 #elif defined(CONFIG_ARM)
 #define BOOTEFI_NAME "bootarm.efi"
+#elif defined(CONFIG_X86_RUN_32BIT)
+#define BOOTEFI_NAME "grubia32.efi"
+#elif defined(CONFIG_X86_RUN_64BIT)
+#define BOOTEFI_NAME "grubx64.efi"
 #endif
 #endif
 
-- 
2.15.0

